### PR TITLE
US19542 Primary Link 

### PR DESCRIPTION
**Pull request by tcmacdonald**
Specify primary link color and text-orange as darker orange. Note– the text-orange update is kind of unrelated to US19542 but I discussed with @amplerob and he suggested that we maintain consistency for any orange text, primarily due to the contrast ratio for the lighter orange (see screenshot below for the inconsistency that is being resolved here.

#### Related PRs: 

crdschurch/crds-media#905

---

Media

### CODE REVIEW

File: assets/stylesheets/utilities/_colors.scss [hunk 1]
```diff
@@ -25,5 +25,5 @@
 }
 
 .text-orange {
-  color: $cr-orange;
+  color: $cr-orange-dark;
```
Comment (silasbaur) [inline assets/stylesheets/utilities/_colors.scss:28]: Why was this changed to dark?

Comment (tcmacdonald) [reply assets/stylesheets/utilities/_colors.scss:32]: See the PR description above bro ☝️
